pkgs/desktops/gnome-3/3.20/core/libpeas/default.nix

```diff
@@ -1,13 +1,15 @@
 { stdenv, fetchurl, pkgconfig, intltool, gnome3
-, glib, gtk3, gobjectIntrospection, python, pygobject3
+, glib, gtk3, gobjectIntrospection, python3, python3Packages, ncurses, python2, python2Packages
 }:
 
 stdenv.mkDerivation rec {
   inherit (import ./src.nix fetchurl) name src;
 
+  configureFlags = [ "--enable-python3" "--enable-python2" ];
+
   buildInputs =  [
-   intltool pkgconfig glib gtk3 gobjectIntrospection python pygobject3
-   gnome3.defaultIconTheme
+   intltool pkgconfig glib gtk3 gobjectIntrospection python3 python3Packages.pygobject3 python2
+   gnome3.defaultIconTheme ncurses
```

Comment:
> /nix/store/0xskayad86h730x6qaqzlyy8j8kalzlb-libpeas-1.18.0
+--- [...]
+---/nix/store/5wyc68zp1qj05ac5xgp18v8qpf259m96-python-2.7.11-env
|   +---/nix/store/dvq0mwb7ic8n6kxrqiykssqjq31gc3g4-pygobject-3.20.0
|   |   +--- [...]
+---/nix/store/nym478rhdc2ljk1pqfgic8cs7qh6a2wy-gobject-introspection-1.48.0 [...]
+---/nix/store/w9gwivwr7ih57b4y1dfhv64xy316p5qn-python3-3.5.1
[...]

This is indeed wrong. It includes the Python 2 env, as we expected, but with Python 3 only the interpreter, and not the environment. That is *very* strange. Was python3 somehow bind to the interpreter in the let expression?

> $ nix-store --query --references /nix/store/3l1kj7wpby7gy5appwq99yc6bmfc4n5l-libpeas-1.18.0.drv 
/nix/store/r13sd830c4576wr60n7inxl66qb7rn80-gobject-introspection-1.48.0.drv
/nix/store/m4dgc7jmg8mz85c4s13gh85icgac1r50-python3-3.5.1-env.drv
/nix/store/qq78rnjlcpj2g02gms4cjl2w68n6jd9i-python-2.7.11-env.drv

Here it does look good...